src/core/server/integration_tests/saved_objects/migrations/group2/check_registered_types.test.ts

```diff
@@ -141,7 +141,7 @@ describe('checking migration metadata changes on all registered SO types', () =>
         "synthetics-param": "9776c9b571d35f0d0397e8915e035ea1dc026db7",
```

Comment:
`rule-settings` looks pretty "fat" - can we remove anything here?  I wouldn't be surprised if we didn't need any of these fields mapped: https://github.com/elastic/kibana/blob/main/x-pack/plugins/alerting/server/saved_objects/rules_settings_mappings.ts

Reply:
Yes, I forget on this one let me check what we do here

Reply:
Looking at the code we do not need it, let me try it. Good catch!